supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc

```diff
@@ -31,6 +31,17 @@
 
 *See also*:
 
+[discrete]
+[[weak-dependencies]]
+==== image:images/yes.png[yes] Weak dependencies (noun)
+*Description*: In Red Hat Enterprise Linux, _Weak dependencies_ are variants of the Requires directive that describe dependencies that are not necessary for the package to work but are considered useful by the package author. You can think of them as best served with.
```

Comment:
```suggestion
*Description*: In Red Hat Enterprise Linux, _Weak dependencies_ are variants of the `Requires` directive that describe dependencies that are not necessary for the package to work but are considered useful by the package author.
```
- Is `Requires` a literal? If so, enclose in back ticks
- Omit final sentence. It's redundant, and the tone is too informal.